src/lib/cache.svelte.ts

```
@@ -111,10 +119,19 @@ async function initialize() {
+		members: {},
 		people: {},
 		meals: {},
 	})
 	
+	// Members
+	
+	const {data: membersData, error: membersError} = await supabase
+		.from("members")
+		.select("household, user, email")
+	if (membersError) { console.error("Error in getting members:", membersError); toast.error("Error in getting members.") }
+	else membersData.forEach(member => households[member.household].members[member.user] = member)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)